packages/agreement-process/test/suspendAgreement.test.ts

```diff
@@ -62,179 +55,12 @@ describe("suspend agreement", () => {
     vi.useRealTimers();
   });
 
-  it("should succeed when requester is Consumer or Producer, Consumer has all attributes satisfied, and the Agreement is in an suspendable state", async () => {
```

Comment:
No need to set valid attributes anymore to suspend the agreement by consumer or producer, see PR description